src/simulator/halo_simulator.py

```diff
@@ -38,6 +40,10 @@
 from wfa_cardinality_estimation_evaluation_framework.estimators.estimator_noisers import (
     GeometricEstimateNoiser,
 )
+from wfa_cardinality_estimation_evaluation_framework.estimators.any_sketch import (
```

Comment:
Is this needed?

Reply:
Nope. It's for noise adding. Do it too early. 
Thanks for catching that!